app/templates/partials/guidance.html

```diff
@@ -0,0 +1,17 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% for item in guidance_content.content %}
+  {%- if item.title -%}
+    <h3 class="venus">{{item.title}}</h3>
+  {% endif %}
+  {%- if item.description -%}
+    <h2 class="mars">{{item.description}}</h2>
```

Comment:
This isn't a heading so `h2` isn't semantically correct. Looking at the schema, there's `p` tags in there so a `div` is the best choice here.